lib/node_modules/@stdlib/math/base/special/asin/src/main.c

```diff
@@ -57,6 +57,7 @@ static const double MOREBITS = 6.123233995736765886130e-17; // pi/2 = PIO2 + MOR
 */
 static double rational_pq( const double x ) {
```

Comment:
I had to rerun `asin/scripts/evalrational.js` to resolve assignment of read-only `x`.